bin/jmeter.properties

```diff
@@ -604,8 +604,15 @@ upgrade_properties=/bin/upgrade.properties
 # it assumes that the user has clicked a new URL
 #proxy.pause=5000
 
-# Add numeric suffix to Sampler names (default true)
+# Add numeric to Sampler names (default true)
```

Comment:
Some placeholder has to be used (in my opinion) instead of *suffix* or use *number* instead of *numeric*.

Reply:
I remove the suffix (add to the JMeter 5.2) because is not only suffix.
I don't change the comment "numeric". This comment exist for a long time. 
I can change to numbering.